src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,265 @@
+                    ddlDscp.substring(splitPoint + 4));
+            taskList.add(newDdl);
+            writeToFile(initializeTextToWrite());
+            printAddSuccessMessage(newDdl);
+        } else {
+            throw new DukeException("Illegal Deadline prompt detected.");
+        }
+    }
+
+    private static void addEventToList(String description) throws DukeException, IOException {
+        if (description.startsWith(" ")){
+            String evtDscp = description.substring(1);
+            int splitPoint = evtDscp.indexOf("/at");
+            if (splitPoint==-1){
+                throw new DukeException("Illegal Event prompt detected.");
+            }
+            Event newEvt = new Event(evtDscp.substring(0, splitPoint - 1),
+                    evtDscp.substring(splitPoint + 4));
+            taskList.add(newEvt);
+            writeToFile(initializeTextToWrite());
+            printAddSuccessMessage(newEvt);
+        } else {
+            throw new DukeException("Illegal Event prompt detected.");
+        }
+    }
+
+    private static void addTodoToList(String description) throws DukeException, IOException {
+        if (description.startsWith(" ")) {
+            Task newTodo = new Todo(description.substring(1));
+            taskList.add(newTodo);
+            writeToFile(initializeTextToWrite());
+            printAddSuccessMessage(newTodo);
+        } else {
+            throw new DukeException("Illegal Todo Prompt detected.");
+        }
+    }
+
+    private static void printAddSuccessMessage(Task task) {
+        System.out.print(DIVLINE);
+        System.out.println("\tGot it. I've added this task:\n" +
+                "\t  " + task.toString());
+        System.out.println("\tNow you have " + taskList.size() + " tasks in the list.");
+        System.out.print(DIVLINE);
+    }
+
+    private static void printDeleteSuccessMessage(Task task) {
+        System.out.print(DIVLINE);
+        System.out.println("\tNoted. I've removed this task: \n" +
+                "\t  " + task.toString());
+        System.out.println("\tNow you have " + taskList.size() + " tasks in the list.");
+        System.out.print(DIVLINE);
+    }
+
+    private static void completeTask(int taskIndex) throws DukeException {
+        if (taskList.get(taskIndex).getIsDone()){
+            throw new DukeException("Task has already been marked done.");
+        }
+        taskList.get(taskIndex).markAsDone();
+        System.out.print(DIVLINE);
+        System.out.println("\tNice! I've marked this task as done: \n" +
+                "\t" + taskList.get(taskIndex).toString());
+        System.out.print(DIVLINE);
+    }
+
+    private static void deleteFromList(int taskIndex) throws DukeException, IOException {
+        if (taskIndex >= taskList.size()) {
+            throw new DukeException("Task does not exist");
+        }
+        Task toBeDeleted = taskList.get(taskIndex);
+        // Some manipulation here
```

Comment:
This comment is a little ambiguous. You could consider describing the below lines a little more. Alternatively, leaving it could be better.